aws-logs-loggroup/pom.xml

```diff
@@ -43,7 +43,7 @@
         <dependency>
             <groupId>software.amazon.awssdk</groupId>
             <artifactId>cloudwatchlogs</artifactId>
-            <version>2.10.49</version>
+            <version>2.13.18</version>
```

Comment:
[documenting this again somewhere publicly accessible](https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-gamelift/pull/7/commits/c7bebe4f72ab6e4816799ca5d5333c0c0f4e7683)

also had to bump my `awssdk` version when upgrading to `aws-cloudformation-rpdk-java-plugin 2.0.0+` due to:
`java.lang.NoSuchFieldError: TLS_TRUST_MANAGERS_PROVIDER`

but couldn't bump it higher than the `awssdk` version in the `java-plugin` itself due to `PROFILE_FILE`:
https://github.com/aws-cloudformation/cloudformation-cli-java-plugin/pull/267#issuecomment-632796555